src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -3,46 +3,154 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-# pylint: disable=line-too-long
-import os
+# pylint: disable=line-too-long,too-many-statements
+
+import os.path
 import platform
 
 from argcomplete.completers import FilesCompleter
-
-from azure.cli.core.commands import register_cli_argument, register_extra_cli_argument
+from azure.cli.core.commands.parameters import file_type
+from azure.cli.core.commands.parameters import get_enum_type
+from azure.cli.core.commands.parameters import get_resource_name_completion_list
+from azure.cli.core.commands.parameters import name_type
```

Comment:
These can all be consolidated into a single line
```Python
from azure...parameters import (
  file_type, name_type, get_enum_type, get_resource_name_completion_list)
```

Reply:
I've always followed the [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) recommendation of `Imports should be on separate lines`, but if this style is preferred locally, that's fine with me.